client/client.go

```diff
@@ -118,13 +118,14 @@ func (c *client) Start(ctx context.Context, execPath string, opts ...OpOption) (
 	ret.applyOpts(opts)
 
 	req := &rpcpb.StartRequest{
-		NetworkId:       ret.networkID,
-		ExecPath:        execPath,
-		NumNodes:        &ret.numNodes,
-		ChainConfigs:    ret.chainConfigs,
-		UpgradeConfigs:  ret.upgradeConfigs,
-		SubnetConfigs:   ret.subnetConfigs,
-		FreshStakingIds: ret.freshStakingIds,
+		NetworkId:              ret.networkID,
+		ExecPath:               execPath,
+		NumNodes:               &ret.numNodes,
+		ChainConfigs:           ret.chainConfigs,
+		UpgradeConfigs:         ret.upgradeConfigs,
+		SubnetConfigs:          ret.subnetConfigs,
+		ZeroIpIfPublicHttpHost: ret.zeroIPIfPublicHttpHost,
```

Comment:
bad name imho. PublicHttpHost is better

Reply:
PublicHttpHost is also a bad name. It makes me thinks it is asking the node to be public.
This is not the case. It is asking the node to show his IP as 0.0.0.0 when it is public.
Will call it ZeroIp so the reader can understand it is something unusual.